Rocket.Chat/Controllers/Auth/AuthViewControllerLoginServices.swift

```diff
@@ -50,12 +50,6 @@ extension AuthViewController {
     }
 
     func setupLoginServices() {
-        self.loginServicesToken?.invalidate()
-
-        self.loginServicesToken = LoginServiceManager.observe { [weak self] changes in
-            self?.updateLoginServices(changes: changes)
-        }
```

Comment:
If we do this, we break Custom OAuth methods since they are not contained in public settings. Can you undo these changes?

You can confirm this by connecting to my server: cardoso.rocket.chat

It has two custom OAuth methods: GITHUB and OPEN

Reply:
@cardoso I looked carefully at this when removing the code. We had a redundancy in the way we were adding the Custom OAuth buttons to the screen that just doesn't make any sense.

If the API works correctly it will add the Custom OAuth buttons to the screen just as it used to do with this duplicated logic. The issue here is that `public settings` request for `cardoso.rocket.chat` is returning `false` for GitHub OAuth:

```
     "type" : "boolean",
      "value" : false,
      "_id" : "Accounts_OAuth_Github"
``` 

I updated the code to rely on `public settings` REST request because it doesn't make sense to have these redundancy ways to handle the rendering of Custom OAuth buttons. It's weird because if you test with `open.rocket.chat` it works, the `Accounts_OAuth_Github` (and others) returns true and its button is rendered on the screen. 

Now I'm not sure if we should rely on `public settings` REST request at all. Because if it can behave this inconsistently at the point of returning `false` for a field that should be `true` maybe it's not stable enough and can cause this kind of issue for other fields too.